src/forms-opportunities/guidance-handlers/guidance-accessibility.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { ok } from '@adobe/spacecat-shared-http-utils';
+
+export default async function handler(message, context) {
+  const { log, dataAccess } = context;
+  const { auditId, siteId, data } = message;
+  const { opportunityId, accessibility: a11yGuidanceOfIssues } = data;
```

Comment:
can you pls confirm the contract here with M. From the M PR, it seems it is `a11y`

Reply:
yes, let me update. 
(Initially thought to put `accessibility` but in M everything is `a11y` so kept the same)
```
data: {
    "opportunityId": "<id>",
    "a11y' : []
}
``

Reply:
but can't M already pull, that data from using the opportunity API using oppoty id?